common/utils/src/main/resources/error/error-classes.json

```diff
@@ -3748,6 +3748,12 @@
     },
     "sqlState" : "0A000"
   },
+  "UNSUPPORTED_ENCODER" : {
+    "message" : [
+      "Found unsupported encoder. Try switching to expression encoder."
```

Comment:
nit:
```suggestion
      "Found unsupported encoder. Try switching to an expression encoder."
```

Reply:
I think we should help users and point out how to switch to an expression encoder. At least, let's point out the doc:
https://spark.apache.org/docs/latest/api/java/index.html?org/apache/spark/sql/Encoder.html

Reply:
Done.